packages/types/src/errors/grid_client/workloads.ts

```diff
@@ -0,0 +1,38 @@
+import { BaseError } from "../base_error";
+import { ErrorModules } from "../modules";
+
+export enum Errors {
+  WorkloadDeleteError,
+  WorkloadDeployError,
+  WorkloadUpdateError,
+  WorkloadCreateError,
```

Comment:
I noticed that when we use this it be like ``` GridClientErrors.Workloads.WorkloadDeleteError```  .Maybe change the naming to DeleteError, DeployError ... etc.

Reply:
error name will not be clear:
![Screenshot from 2023-11-22 12-44-07](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/b9f8e9bf-176f-40bb-bf6c-968e6b2db88e)

in this case the error name appears as `RequestError`, the current error name in case of workloads will be `WorkloadDeleteError` if we delete the workload from error name will not be that clear,
also the generic will be workload but that not make the error message that clear 
what do u think ?